### DISCUSSION TITLE
[Feature] cannot upload local file to shared album on mobile app

### DISCUSSION THREAD

**Discussion by he2121** [immich-app/immich#13057] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I loves the feature of "shared album", it is great that I can share my photos to my friends. 
However, It seems to can not upload the local photos to the shared album on mobile app. I do not want to upload all of my photos to the server, it is so awkward... 
And I found that I can upload local photos to shared album on Web...

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#12190
